test/PriceOracle.test.ts

```diff
@@ -82,6 +83,9 @@ describe('PriceOracle', function () {
         await priceOracle.setPrice(minPriceString);
         price0SetBlock = await getBlockNumber();
 
+        // Set price base
+        const priceBase = await priceOracle.priceBase();
```

Comment:
This issue reported by the checker should be addressed.

Reply:
fixed